third_party/brotli/c/dec/huffman.c

```diff
@@ -8,12 +8,16 @@
 
 #include "./huffman.h"
 
-#include <string.h>  /* memcpy, memset */
```

Comment:
Lets leave it in the original place, if needed bracket it with #if !defined(STARBOARD)

But i don't think excluding is is needed even anymore.